packages/manager/src/constants/API_ENDPOINTS.ts

```diff
@@ -41,14 +41,29 @@ export const API_ENDPOINTS: APIEndpoints = (() => {
 
 			if (missingAPIEndpoints.length > 0) {
 				console.error(
-					`You are running Slice Machine in development mode where API endpoints are configured via environment variables. The following endpoints were not configured: ${missingAPIEndpoints.join(
-						", ",
-					)}. Configure them before continuing.`,
+					`You are running Slice Machine in development mode (SM_ENV=${
```

Comment:
I have a suggestion outside the scope of this PR: `@slicemachine/manager` is a Node.js library, not a CLI. Rather than calling `console.error`, maybe we should be throwing errors instead. Then, in a CLI like `@slicemachine/init`, we could catch these errors and log them to the console.

An exception to this might be the `console.warn` below since we don't want to accidentally not warn users about potentially malicious endpoints.

Reply:
Thinking about it twice, I'm not sure how another package could catch such error since the error would be thrown upon import of the manager because `API_ENDPOINTS.ts` is not a function we run, it's just executed upon load.

I think we'll leave that out for now as I don't see a simple path towards something like that now 🤔